examples/docker-for-mac/my_test.sh

```diff
@@ -21,15 +21,24 @@ set -o pipefail
 readonly IMAGE_TAG=v2.2.0
 readonly IMAGE_REPOSITORY="quay.io/helmpack/chart-testing"
 
-main() {
-    local testcontainer_id
-    testcontainer_id=$(create_testcontainer)
+run_ct_container() {
+    echo 'Running ct container...'
+    docker container run --rm --interactive --detach --name ct \
```

Comment:
You should leave the existing behavior and not use a hard-coded container name. Otherwise you won't be able to do parallel executions.